ngrinder-controller/src/main/java/org/ngrinder/script/controller/FileEntryApiController.java

```diff
@@ -65,6 +66,7 @@
 @RestController
 @RequestMapping("/script/api")
 @RequiredArgsConstructor
+@PreAuthorize("hasAnyRole('A', 'S', 'U')")
```

Comment:
All of FileEntiry related APIs need authorization even if the method doesn't have a User parameter.